### PR TITLE
Hardware selection guide

### PR DESCRIPTION
**Pull request by felix920506**
Followup to #655 

A guide on selecting hardware.
Still requires linking from various places.
However since this is going to require extensive feedback from all parties I am creating the PR early on.

### CODE REVIEW

File: docs/general/administration/hardware-selection.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+### Upgrading Existing System for use as a Server
+
+We recommend Intel ARC GPUs for upgrading existing systems. If Intel ARC GPUs aren't available, we recommend Nvidia GTX 16 / RTX 20 or newer, excluding the GTX 1650.
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, Notably:
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:40]:
```suggestion
The CPU will be responsible for many tasks, notably:
```

File: docs/general/administration/hardware-selection.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).
+
+## TL;DR
+
+For a Jellyfin server, we recommend the following:
+
+### Normal Server From Scratch
+
+- CPU: 4 Threads
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:17]: Where is this coming from?

File: docs/general/administration/hardware-selection.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (ex. Hardware Acceleration).
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:8]:
```suggestion
This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g.. hardware acceleration).
```

File: docs/general/administration/hardware-selection.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+## TL;DR
+
+For a Jellyfin server, we recommend the following:
+
+### Normal Server From Scratch
+
+- CPU: 4 Threads
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:16]:
```suggestion
- CPU: 4 threads
```

File: docs/general/administration/hardware-selection.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  - Please Avoid 11th Gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
+  - Please Avoid Rapsberry PIs and other low powered SBCs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:18]:
```suggestion
  - Please avoid Rapsberry PIs and other low powered SBCs.
```

File: docs/general/administration/hardware-selection.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  - Please Avoid Intel Core M / Y series CPUs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:19]:
```suggestion
  - Please avoid Intel Core M / Y series CPUs.
```

File: docs/general/administration/hardware-selection.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+- RAM: 8GB or more
+- Storage: SSD for Jellyfin files and transcoding cache.
+- Graphics: Intel > Nvidia > AMD > Apple.
+  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
+  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:24]: Might be good to add an infobox about the reasoning behind this.

File: docs/general/administration/hardware-selection.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+  - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:25]: Is not true for AV1 on AMD anymore, just an FYI

File: docs/general/administration/hardware-selection.md (hunk 2 above)
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:17]:
```suggestion
  - Please Avoid 11th gen or older Intel Atom and Celeron / Pentium N / J series CPUs.
```

File: docs/general/administration/hardware-selection.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+We recommend Intel ARC GPUs for upgrading existing systems. If Intel ARC GPUs aren't available, we recommend Nvidia GTX 16 / RTX 20 or newer, excluding the GTX 1650.
+
+## Detailed Guide
+
+### CPU
+
+The CPU will be responsible for many tasks, Notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:43]:
```suggestion
- Decoding video in unsupported codecs
```

File: docs/general/administration/hardware-selection.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:45]: I don't think this is true if source is 4k HDR

File: docs/general/administration/hardware-selection.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
+#### Resizable BAR
+
+Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.
+
+When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
+
+### System Memory (RAM)
+
+As a general rule of thumb, we recommend at least 8GB for Linux and 16GB for Windows, though significantly smaller amounts may work just fine.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:59]: Why double it for Windows?

File: docs/general/administration/hardware-selection.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the hardware acceleration options available is entirely determined by the choice of GPUs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:63]:
```suggestion
With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
```

File: docs/general/administration/hardware-selection.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+
+The following is a list of commonly used codecs:
+
+- H.264 (AVC): Blu-Ray, most streaming services
+- H.265 (HEVC): Blu-Ray UHD
+- MPEG-2: DVD
+- VC-1: Blu-Ray
+- VP8, VP9, AV1: streaming services
+
+The following is a list of codecs Jellyfin supports transocding to:
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:75]: In theory not correct, we could transcode to others too. Just a note though.

File: docs/general/administration/hardware-selection.md [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
+
+#### Intel Graphics
+
+Please refer to [This Table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:84]:
```suggestion
Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
```

File: docs/general/administration/hardware-selection.md [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+
+Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.
+
+#### Nvidia Graphics
+
+Please refer to [This Table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:90]:
```suggestion
Please refer to [this table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
```

File: docs/general/administration/hardware-selection.md [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+
+#### AMD Graphics
+
+Please refer to [This Article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:94]:
```suggestion
Please refer to [this article](https://en.wikipedia.org/wiki/Video_Coding_Engine) for supported codecs.
```

File: docs/general/administration/hardware-selection.md [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+
+#### Apple Silicon
+
+Supported codecs are listed below:
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:98]: Might want to add 10bit HEVC and VP9 capabilities

File: docs/general/administration/hardware-selection.md [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+
+Refer to the Pro column for Max and Ultra SOCs.
+
+✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.
+
+Please note that many hardware acceleration features are not available on MacOS for Jellyfin, and no Apple Silicon media engine drivers exist for other operating systems.
+
+### Storage
+
+There are 2 general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:117]:
```suggestion
There are two general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```

File: docs/general/administration/hardware-selection.md [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+
+#### Media Storage
+
+Anything with higher sequential access speed than bitrate of media will work.
+
+#### Jellyfin Files
+
+Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience. (But if you want your JF home page take 10 mins to load then sure.)
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:125]:
```suggestion
Jellyfin files will see lots of random access, therefore we recommend SSDs for Jellyfin files. AVOID mechanical drives that use SMR (Shingled Magnetic Recording) as they will result in a VERY poor experience.
```

Kinda unnecessary snarky comment even though it it true lol

File: docs/general/administration/hardware-selection.md (hunk 7 above)
Comment (srcrist) [reply docs/general/administration/hardware-selection.md:24]: I also think this could be phrased a little bit better. This is unclear, and I think the language should be made less direct.

"You should consider Intel ARC over Nvidia" vs "We recommend Intel ARC over Nvidia"

This language would be better throughout.

File: docs/general/administration/hardware-selection.md [hunk 20]
```diff
@@ -14,16 +14,16 @@ For a Jellyfin server, we recommend the following:
...
 - Storage: SSD for Jellyfin files and transcoding cache.
 - Graphics: Intel > Nvidia > AMD > Apple.
-    - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
-    - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
-    - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
-    - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.
+  - We recommend a system with Intel 7th gen or newer Core series CPU with integrated graphics.
+  - When using dedicated graphics, we recommend Intel ARC over Nvidia GTX 16 / RTX 20 or newer GPUs, excluding the GTX 1650.
+  - AMD Graphics or Apple Silicon is NOT recommended due to poor encoder quality.
+  - When Intel based Mac hardware is used, we recommend installing Windows or Linux for the best experience.
```
Comment (srcrist) [inline docs/general/administration/hardware-selection.md:26]: As above: "is recommended" over "we recommend" here is better. It might also be worth thinking of framing it in terms of WHY it might be recommended, instead of just saying that X or Y is recommended. For example: "When using Intel based Mac hardware, installing Windows or Linux enables features that Mac OS does not, and will better support transcoding and hardware acceleration."

File: docs/general/administration/hardware-selection.md (hunk 19 above)
Comment (srcrist) [reply docs/general/administration/hardware-selection.md:125]: Agreed. Official documentation pages need to keep an air of impartiality and professionalism. The project is represented by these pages.

File: docs/general/administration/hardware-selection.md [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+The CPU will be responsible for many tasks, Notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs.
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
+
+#### Integrated Graphics
+
+Integrate graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
```
Comment (srcrist) [inline docs/general/administration/hardware-selection.md:49]:
```suggestion
Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
```

File: docs/general/administration/hardware-selection.md [hunk 22]
```diff
@@ -0,0 +1,125 @@
...
+
+As a general rule of thumb, we recommend at least 8GB for Linux and 16GB for Windows, though significantly smaller amounts may work just fine.
+
+### Graphics Cards (GPUs)
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the hardware acceleration options available is entirely determined by the choice of GPUs.
+
+We recommend Intel graphics over Nvidia graphics because of the quality. We recommend avoiding AMD or Apple Silicon since they produce poor quality output. Encoder Quality: Intel > Nvidia > AMD > Apple
+
+The following is a list of commonly used codecs:
```
Comment (srcrist) [inline docs/general/administration/hardware-selection.md:67]: I honestly think this entire section is unnecessary. Or at least poorly placed. It very well be nice to have in the documentation somewhere, but I'm not sure that it belongs in a hardware guide.

Comment (Shadowghost) [reply docs/general/administration/hardware-selection.md:67]: We already have it in the documentation somewhere.

File: docs/general/administration/hardware-selection.md (hunk 2 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: 4 threads is what I think should be able to comfortably handle most software decoding tasks in legacy formats.
Atom, Celeron/Pentium N/J series was excluded because they are low powered parts that may struggle in some tasks but correct me if I'm wrong here.

File: docs/general/administration/hardware-selection.md (hunk 8 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:25]: I'll leave a note

File: docs/general/administration/hardware-selection.md (hunk 10 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:45]: That's fair, I will add some clarification

File: docs/general/administration/hardware-selection.md (hunk 11 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:59]: Windows itself as an OS is more bloated so I don't think it's a bad idea to have more.
It's true that windows with 8GB will work perfectly for a JF server but it can get tight.

File: docs/general/administration/hardware-selection.md (hunk 22 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:67]: Are you referring to the client codec support page?

File: docs/general/administration/hardware-selection.md (hunk 20 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:26]: This section is meant to be a TL;DR so I think the explanation should live somewhere else.

File: docs/general/administration/hardware-selection.md (hunk 17 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:118]: This info isn't published by Apple and the info here was pieced together with info from the internet
There are people who tested VP9 to be supported but I don't think anyone tested 10bit VP9.
I have an M1 Pro device on hand so I can test it out on my system for this particular chip if there are test clips available.

File: docs/general/administration/hardware-selection.md (hunk 7 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: Addressed in a commit

File: docs/general/administration/hardware-selection.md (hunk 17 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:118]: I've done some testing and VP9 10bit does hardware decode on my M1 Pro Macbook

File: docs/general/administration/hardware-selection.md (hunk 22 above)
Comment (Shadowghost) [reply docs/general/administration/hardware-selection.md:87]: Correct

Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: I have changed that table to this link.

Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: While I do agree that the codec section would be better off somewhere else, I do think that having a minimal section here until that codec page exists would be very helpful

**Comment by felix920506** [comment]
I think I have addressed most comments above, please re-review.

**Comment by felix920506** [comment]
After some testing I found Apple Silicon to have at least partial hardware acceleration support for 10bit H264 decoding so I marked them as Decode only.
I also removed ProRes as they are irrelevant for a media server

File: docs/general/administration/hardware-selection.md [hunk 23]
```diff
@@ -0,0 +1,158 @@
...
+
+:::note A Note on Apple hardware
+
+It is recommended that Intel-based Macs be used with Windows or Linux installed to host Jellyfin. Many hardware acceleration features aren't available on MacOS due to the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork not being available.
+
+:::
+
+### Low Power Applications
+
+For low power applications, Intel 12th gen or newer Atom CPUs with integrated graphics are recommended.
```
Comment (JPVenson) [inline docs/general/administration/hardware-selection.md:36]: Link to the LowPower settings would be nice

File: docs/general/administration/hardware-selection.md [hunk 24]
```diff
@@ -0,0 +1,158 @@
...
+
+With modern systems, the media engines are usually located on the GPUs. Therefore, the available hardware acceleration options are determined by the choice of GPU.
+
+Encoder Quality: Intel > Nvidia > AMD > Apple
+
+Intel graphics is recommended over Nvidia graphics because of the quality. AMD or Apple Silicon are not recommended since they produce poor quality H.264 and H.265(HEVC) output.
+
+:::note A Note on AMD AV1 Encoders
+
+With AV1, AMD has significantly improved the quality of their encoders. However, transcoding to AV1 is currently not supported on Jellyfin, and transcoding to H.264 or H.265(HEVC) is still more likely due to AV1 being in the early stages of adoption.
```
Comment (JPVenson) [inline docs/general/administration/hardware-selection.md:83]: To give present accurate description i would rephrase to:
`[...]With jellyfin server version 10.8.10, AV1 codec is not supported[...]`
here instead of below

File: docs/general/administration/hardware-selection.md [hunk 25]
```diff
@@ -0,0 +1,158 @@
...
+A list of common codecs can be found [here](/docs/general/clients/codec-support/)
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+:::note AV1 Support
+
+Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this will change in the near future.
```
Comment (JPVenson) [inline docs/general/administration/hardware-selection.md:96]: `might change in the future`

**Review by JPVenson — CHANGES_REQUESTED** [review]

File: docs/general/administration/hardware-selection.md [hunk 26]
```diff
@@ -132,16 +136,20 @@ Refer to the Pro column for Max and Ultra SOCs.
 
 ✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.
 
-Please note that many hardware acceleration features are not available on MacOS for Jellyfin, and no Apple Silicon media engine drivers exist for other operating systems.
+:::caution
+
+Many hardware acceleration features are not available on MacOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for MacOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:135]:
```suggestion
Many hardware acceleration features are not available on macOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for macOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).
```

File: docs/general/administration/hardware-selection.md [hunk 27]
```diff
@@ -103,7 +103,11 @@ For decoding support, the more codecs the better. However, there are only a few
 
 Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
 
-Do Note that Intel ARC requres kernel version 6.2 or newer on Linux. Please check your distribution for details.
+:::note Intel 12th Gen, 13th Gen and ARC Support
+
+If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that jellyfin-ffmpeg6 be installed instead of the default jellyfin-ffmpeg5 on these hardware platforms.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:103]:
```suggestion
If you are planning to use Linux with Intel 12/13th Gen integrated graphics or ARC, these GPUs only work on Linux Kernel 6.2 or newer. Please check your distribution to make sure it has a supported Linux Kernel version. It is also recommended that `jellyfin-ffmpeg6` be installed instead of the default `jellyfin-ffmpeg5` on these hardware platforms.
```

File: docs/general/administration/hardware-selection.md (hunk 2 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: This line no longer exists in PR and I will close this

File: docs/general/administration/hardware-selection.md (hunk 23 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:29]: Where is this located in the docs?

**Comment by felix920506** [comment]
I have addressed all above comments except the lowpower docs which I can't find.

File: docs/general/administration/hardware-selection.md (hunk 23 above)
Comment (JPVenson) [reply docs/general/administration/hardware-selection.md:29]: https://jellyfin.org/docs/general/administration/hardware-acceleration/intel#low-power-encoding

**Comment by Shadowghost** [comment]
If you mean the Intel low power docs, they are in here: https://jellyfin.org/docs/general/administration/hardware-acceleration/intel

File: docs/general/administration/hardware-selection.md (hunk 23 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: Linked

**Comment by felix920506** [comment]
I have linked the low power encoding docs as well. What's remaining is to link this page in all sensible places in the docs.

**Review by JPVenson — APPROVED** [review]

File: docs/general/administration/hardware-selection.md [hunk 28]
```diff
@@ -0,0 +1,177 @@
...
+### Normal Server
+
+- CPU: Intel Core i3-2100, AMD FX-4100 or better
+- RAM: 8GB or more
+- Storage: 32GB SSD storage for Jellyfin files and transcoding cache.
+- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel recommended over Nvidia, AMD and Apple Silicon not recommended.
+
+:::tip Upgrading an Existing System
+
+Intel ARC GPUs are recommended when upgrading an existing system to be used as a Jellyfin server. If cases where Intel ARC is not feasible, Nvidia graphics is recommended. Please select an appropriate GPU according to the recommended specs above.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:23]:
```suggestion
Intel ARC GPUs are recommended when upgrading an existing system to be used as a Jellyfin server. In cases where Intel ARC is not feasible, Nvidia graphics is recommended. Please select an appropriate GPU according to the recommended specs above.
```

File: docs/general/administration/hardware-selection.md [hunk 29]
```diff
@@ -0,0 +1,177 @@
...
+The CPU will be responsible for many tasks, notably:
+
+- Transcoding Audio
+- Decoding video in unsupported codecs
+
+Audio codecs are very lightweight and most CPUs should be able to handle them without issues. Video in unsupported codecs are usually older formats that are easier to decode. Assuming hardware acceleration is properly configured, any modern CPU with 4 threads should be able to handle the workload.
+
+:::caution
+
+Newer codecs can also require software decoding if there are no available hardware decoders. High bitrate HEVC content such as commonly found in 4k HDR can be very demanding even on modern CPUs. Please considering adding a modern GPU with 10bit HEVC decoding support.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:50]: Tone-mapping is also a problem (without hwa this is currently not possible)

File: docs/general/administration/hardware-selection.md [hunk 30]
```diff
@@ -0,0 +1,177 @@
...
+
+:::
+
+#### Integrated Graphics
+
+Integrated graphics can be useful for transcoding video. Please refer to the GPU section for a guide.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:56]: Add a link to the section

File: docs/general/administration/hardware-selection.md [hunk 31]
```diff
@@ -0,0 +1,177 @@
...
+
+#### Resizable BAR
+
+Resizable Bar (ReBAR) is a PCI Express feature that requires support from the platform. This feature is supported on Intel 10th gen or above and AMD Ryzen 3000 Series or above.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:60]: Maybe a short explanation what it is and why it helps is useful

File: docs/general/administration/hardware-selection.md [hunk 32]
```diff
@@ -0,0 +1,177 @@
...
+
+When using Intel ARC Graphics, Resizable BAR is recommended. Disabling it will result in a 10% reduction in transcoding performance. However, given how fast the media engine on Intel ARC is, this will not be a problem for the vast majority of users.
+
+### System Memory (RAM)
+
+As a general rule of thumb, at least 8GB is recommended, though significantly smaller amounts may work just fine. If integrated graphics is being used, please consider adding more as integrated graphics usually use part of the system memory as video memory.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:66]: Note on dual channel memory might be useful here (see hwa guide)

File: docs/general/administration/hardware-selection.md [hunk 33]
```diff
@@ -0,0 +1,177 @@
...
+A list of common codecs can be found [here](/docs/general/clients/codec-support/)
+
+The following is a list of codecs Jellyfin supports transocding to:
+
+- H.264 (Most common transcode target)
+- H.265 (Limited supported by clients)
+
+:::note AV1 Support
+
+Jellyfin currently (as of 10.8.10) does NOT support transcoding to AV1 as a target. Source content in AV1 will transcode to other codecs normally if needed. However, this might change in the future.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:91]: The last sentences seems to reference the sentence before it but should the first sentence. The decoding capabilities will not change

File: docs/general/administration/hardware-selection.md [hunk 34]
```diff
@@ -0,0 +1,177 @@
...
+
+For decoding support, the more codecs the better. However, there are only a few codecs that media is commonly available in, as listed above.
+
+#### Intel Graphics
+
+Please refer to [this table](https://en.wikipedia.org/wiki/Intel_Quick_Sync_Video#Hardware_decoding_and_encoding) for supported codecs.
+
+:::caution F series CPUs
+
+Intel CPUs with a model that ends in F don't have integrated graphics. These are intended to be used with a dedicated graphics card.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:103]:
```suggestion
Intel CPUs with a model description that ends with F don't have integrated graphics. These are intended to be used with a dedicated graphics card.
```

File: docs/general/administration/hardware-selection.md [hunk 35]
```diff
@@ -0,0 +1,177 @@
...
+
+:::
+
+#### Nvidia Graphics
+
+Please refer to [this table](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) for supported codecs.
+
+:::caution Low End Cards
+
+Certain low end cards (e.g. GT 1030) don't have encoding hardware. Please be careful when choosing a GPU.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:119]: Is the proper name really `GT 1030`?

File: docs/general/administration/hardware-selection.md [hunk 36]
```diff
@@ -0,0 +1,177 @@
...
+
+:::caution Low End Cards
+
+Certain low end cards (e.g. RX 6400, RX 6500) don't have encoding hardware. Please be careful when choosing a GPU.
+
+:::
+
+:::caution Integrated Graphics
+
+Many AMD CPUs don't have integrated graphics. These are intended to be used with a dedicated graphics card. Please check the product page for each CPU for more information.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:135]: Might want to add a note which CPU series include a GPU (e. g. the `G` denoted and Zen4)

File: docs/general/administration/hardware-selection.md [hunk 37]
```diff
@@ -0,0 +1,177 @@
...
+| H.265 8bit  | ✅  | ✅      | ✅  | ✅      |
+| H.265 10bit | ✅  | ✅      | ✅  | ✅      |
+| VP9 8bit    | 🔶  | 🔶      | 🔶  | 🔶      |
+| VP9 10bit   | 🔶  | 🔶      | 🔶  | 🔶      |
+| AV1         | ❌  | ❌      | ❌  | ❌      |
+
+Refer to the Pro column for Max and Ultra SOCs.
+
+✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.
+
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:156]: We might just want to move this to the codecs page and reference it here

File: docs/general/administration/hardware-selection.md [hunk 38]
```diff
@@ -0,0 +1,177 @@
...
+:::caution
+
+Many hardware acceleration features are not available on macOS for Jellyfin, as the custom [jellyfin-ffmpeg](https://github.com/jellyfin/jellyfin-ffmpeg) fork isn't available for macOS. No Apple Silicon media engine drivers exist for other operating systems currently. You will NOT be able to use hardware acceleration if you are running [Asahi Linux](https://asahilinux.org/).
+
+:::
+
+### Storage
+
+There are two general types of storage that will be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:165]:
```suggestion
There are two general types of storage that can be used on a Jellyfin server. Media libraries and Jellyfin files require different performance characteristics.
```

Comment (Shadowghost) [reply docs/general/administration/hardware-selection.md:165]: Or `should` as an alternative

**Review by Shadowghost — COMMENTED** [review]
Please squash the commits before we merge this

File: docs/general/administration/hardware-selection.md (hunk 37 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:156]: I think this can stay here for now as the codec page is intended for client side. I do think that it won't be a bad idea to have a makeover of the codecs page

File: docs/general/administration/hardware-selection.md [hunk 39]
```diff
@@ -0,0 +1,188 @@
...
+## TL;DR
+
+For a Jellyfin server, the following is recommended:
+
+### Normal Server
+
+- CPU (With dGPU): Intel Core i5-2300, AMD Ryzen 3 1200 or better. (Geekbench 6 Multicore 1500 or better)
+- CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better)
+- RAM: 8GB or more
+- Storage: 32GB SSD storage for Jellyfin files and transcoding cache.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:19]: We might want to bump that. 32GB for cache and file might be a bit on the lower side (e.g. trancoding 4k might already eat 20GB)

File: docs/general/administration/hardware-acceleration/intel.md
Comment (Shadowghost) [inline docs/general/administration/hardware-acceleration/intel.md:1]: I think it would be nice if we could somehow preserve the info here. It is technical but for more advanced users this might be helpful.

File: docs/general/administration/hardware-acceleration/amd.md
Comment (Shadowghost) [inline docs/general/administration/hardware-acceleration/amd.md:1]: I think it would be nice if we could somehow preserve the info here. It is technical but for more advanced users this might be helpful.

File: docs/general/administration/hardware-acceleration/nvidia.md
Comment (Shadowghost) [inline docs/general/administration/hardware-acceleration/nvidia.md:1]: I think it would be nice if we could somehow preserve the info here. It is technical but for more advanced users this might be helpful.

File: docs/general/administration/hardware-selection.md [hunk 40]
```diff
@@ -0,0 +1,188 @@
...
+| H.264 10bit | 🔶  | 🔶      | 🔶  | 🔶      |
+| H.265 8bit  | ✅  | ✅      | ✅  | ✅      |
+| H.265 10bit | ✅  | ✅      | ✅  | ✅      |
+| VP9 8bit    | 🔶  | 🔶      | 🔶  | 🔶      |
+| VP9 10bit   | 🔶  | 🔶      | 🔶  | 🔶      |
+| AV1         | ❌  | ❌      | ❌  | ❌      |
+
+Refer to the Pro column for Max and Ultra SOCs.
+
+✅ = Supported, 🔶 = Decode Only, ❌ = Not Supported.
```
Comment (Shadowghost) [inline docs/general/administration/hardware-selection.md:166]: See other comments, might be better suited for an advanced section.

File: docs/general/administration/hardware-acceleration/intel.md
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/intel.md:1]: Restored the expert guide

File: docs/general/administration/hardware-acceleration/nvidia.md
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/nvidia.md:1]: Restored the expert guide

File: docs/general/administration/hardware-acceleration/amd.md
Comment (felix920506) [reply docs/general/administration/hardware-acceleration/amd.md:1]: Restored the expert guide

File: docs/general/administration/hardware-selection.md (hunk 40 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md:166]: I think this can stay here for now for the lack of a better place to put it without pouring hours into writing a dedicated apple silicon guide which realistically isn't recommend for hosting JF anyway.

File: docs/general/administration/hardware-selection.md (hunk 39 above)
Comment (felix920506) [reply docs/general/administration/hardware-selection.md]: I bumped it to 60GB.

**Review by Shadowghost — APPROVED** [review]
